### PR TITLE
Update Helm dependency to v3.4.0

### PR DESCRIPTION
**Pull request by unguiculus**
Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

### CODE REVIEW

File: magefile.go [hunk 1]
```diff
@@ -41,10 +41,14 @@ func Lint() error {
 	if err := sh.RunV("goimports", "-w", "-l", "."); err != nil {
 		return err
 	}
+	if err := sh.RunV("git", "diff", "--exit-code"); err != nil {
+		return err
+	}
+	// Run after git diff check because it produces random changes different from when run locally
```
Comment (scottrigby) [inline magefile.go:47]: 💯

**Review by scottrigby — APPROVED** [review]
lgtm

File: magefile.go (hunk 1 above)
Comment (unguiculus) [reply magefile.go]: Had to change it again. go.mod changes also. Weird.

**Comment by scottrigby** [comment]
Nice find. For future ref, this fix is from https://github.com/ory/dockertest/issues/212

**Pull request merged**